docs/en/04_Changelogs/4.3.0.md

```diff
@@ -41,3 +42,47 @@ public function getCMSFields()
 }
 
 ```
+
+### Keep using the legacy `CMSPageHistoryController`
+
+To keep using the old CMS history controller for every page type, add the following entry to your your YML config.
+
+```yml
+SilverStripe\Core\Injector\Injector:
+  SilverStripe\CMS\Controllers\CMSPageHistoryController:
+    class: SilverStripe\CMS\Controllers\CMSPageHistoryController
+```
+
+Alternatively, you can use `HistoryControllerFactory` to switch between both controllers.
```

Comment:
We deprecated this class so I don’t think we should document how to use it

Reply:
I'm not especially attached to this bit. But if we remove it and someone decides they want to keep exactly the same history controller behavior they had in 4.2, they don't really have away of to find out without reading through our source code.

If you think that's an acceptable compromise, I'm good removing it.

Reply:
What about saying that you may use a factory to generate either of the controllers with your own custom logic:

```yml
SilverStripe\Core\Injector\Injector:
  SilverStripe\CMS\Controllers\CMSPageHistoryController:
    factory:
      Vendor\Package\MyCustomControllerFactory
```

I agree that it's strange to add documentation about something that's being removed, and someone will have to come back and remove this documentation eventually.

Reply:
Good idea. I've added your blurb and linked off to the Factory section on the Injector doc.